assets/src/edit-story/components/inspector/inspectorLayout.js

```diff
@@ -61,14 +62,16 @@ function InspectorLayout() {
   } = useInspector();
   useEscapeToBlurEffect(ref);
   return (
-    <Layout ref={ref}>
-      <TabsArea>
-        <InspectorTabs />
-      </TabsArea>
-      <InspectorBackground ref={setInspectorContentNode}>
-        <InspectorContent />
-      </InspectorBackground>
-    </Layout>
+    <ColorPickerProvider>
+      <Layout ref={ref}>
+        <TabsArea>
+          <InspectorTabs />
+        </TabsArea>
+        <InspectorBackground ref={setInspectorContentNode}>
+          <InspectorContent />
+        </InspectorBackground>
+      </Layout>
+    </ColorPickerProvider>
```

Comment:
This whole thing was the easiest way I could get the color picker to show outside the panels. The panel have `overflow:auto`, due to them being able to scroll, which didn't easily allow outside elements. However, it'll look weird when then color input component scrolls in a large panel, I guess?

Reply:
I made a comment about this at the `overflow:hidden` change. Let's continue discussion there?